CMakeLists.txt

```
@@ -141,7 +154,7 @@ if(GSTREAMER)
 	set(FAUDIO_GSTREAMER_DEPS " gstreamer-app-1.0 gstreamer-audio-1.0")
 
 	# Add the extra definition...
-	target_compile_definitions(FAudio PRIVATE HAVE_GSTREAMER=1)
+	target_compile_definitions(FAudio PRIVATE HAVE_WMADEC=1)
```

Oh wait just realized, this probably has to define _both_, right?

---

Quick ping for
```suggestion
	target_compile_definitions(FAudio PRIVATE HAVE_GSTREAMER=1 HAVE_WMADEC=1)
```
Everything else looks fine so I can merge this right away, and the next release can label this as "experimental" until Wine's ready to go full PE for the XAudio2 implementation.

---

Sure, sorry I've been side tracked a bit. I squashed the fix and updated the PR.